sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -51,18 +61,25 @@ def __init__(self, dataflow_client=None, job_result=None, job_graph=None):
     self._cached_metrics = None
     self._job_graph = job_graph
 
+  @staticmethod
+  def is_counter(metric_result):
+    return isinstance(metric_result.attempted, numbers.Number)
```

Comment:
what number types you are expecting here? Curios, would not something like `isinstance(3, (int, float))` work?